recipe/meta.yaml

```diff
@@ -26,14 +26,18 @@ requirements:
   build:
     - toolchain
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
     - vc 9              # [win and py27]
     - vc 14             # [win and py>=35]
     - boost-cpp {{ version }}
 
   run:
     - python
-    - numpy x.x
+    - numpy >=1.7  # [py27]
+    - numpy >=1.9  # [py35]
+    - numpy >=1.11  # [py36]
```

Comment:
@jjhelmus I got `numpy 1.7.*` working with `openblas` for `py27` but not `py35`. I wonder if we should try `numpy 1.9` everywhere to simplify this. What do you think?